4.JQUERY/ncmyongjavascript.js

```diff
@@ -0,0 +1,63 @@
+/*Part 1*/
+function sumOdd() {
+		var m =0;
+		var sumOdd = 0;
+		for (m=0; m<=50; m++) {
+			if (m%2 !=0){
+				sumOdd+=m;
+			}
+		}alert(sumOdd);
```

Comment:
Please be mindful of new lines next time :). Alert must be on new line for code readability.